docs/endpoints/account/registerGJAccount.md

```diff
@@ -0,0 +1,27 @@
+# Register Account
+
+> This endpoint is used to register an account
+
+## Parameters
+
+| Parameter | Explanation | Optional |
+| --- | --- | --- |
+| `userName` | The username of the account to create | `False` |
+| `password` | The password of the account to create | `False` |
+| `email` | The email of the account to create | `False` |
+| `secret` | Wmfv3899gc9 | `False` |
+
+## Response
+
+**Successful Request**
+
+```
+1
+```
+
+**Failed Request**
+
+| Error Code | Meaning |
+| --- | --- |
+| -2 | Already taken |
+| -4 | Longer than 20 characters |
```

Comment:
There are a couple more error codes.  

Looking at the enum, these are the codes i expect to appear upon getting these errors however, this has not been checked

```cpp
enum AccountError
{
    kAccountErrorGeneric = -1,
    kAccountErrorTakenUsername = -2,
    kAccountErrorTakenEmail = -3,
    kAccountErrorInvalidUsername = -4,
    kAccountErrorInvalidPassword = -5,
    kAccountErrorInvalidEmail = -6,
    kAccountErrorPasswordTooShort = -8,
    kAccountErrorUsernameTooShort = -9,
};```